MaterialDesignThemes.Wpf/Converters/MathMultipleConverter.cs

```diff
@@ -12,11 +12,61 @@ namespace MaterialDesignThemes.Wpf.Converters
     public sealed class MathMultipleConverter : IMultiValueConverter
     {
         public MathOperation Operation { get; set; }
-        
+
+        /// <summary>
+        /// if true, null arguments will be treated as 0;
+        /// default: false
+        /// </summary>
+        public bool TreatNullAsZero { get; set; }
```

Comment:
Are we actually using this, anywhere?  It would become part of the public API, so has to justify itself.